### ISSUE TITLE
[false-positive] `This sealed trait is never extended`

### ISSUE THREAD

**Issue by jrudolph** [HairyFotr/linter#6]
First of all, thanks for maintaining linter! I've ran it over the current spray code base (jrudolph/spray@bc7bbb72bf113ca16fc01ca07b981950c4b4dd00) to see what it catches. I think I've found several issues and I'm putting them into here one by one.

```
[warn] /home/johannes/git/telfish/spray/spray/spray-io/src/main/scala/akka/io/Tcp.scala:91: This sealed trait is never extended.
[warn]   sealed trait Message
```

but it definitely is

``` scala
  sealed trait Message

  /// COMMANDS

  /**
   * This is the common trait for all commands understood by TCP actors.
   */
  trait Command extends Message with SelectionHandler.HasFailureMessage {
    def failureMessage = CommandFailed(this)
  }
```

Not sure what the exact condition is.
